extensions/BugModal/web/bug_modal.css

```diff
@@ -598,7 +598,7 @@ h3.change-name {
 
 .comment-text {
     white-space: pre-wrap;
-    font: 13px/1.2 "Droid Sans Mono",Menlo,Monaco,"Courier New",Courier,monospace;
+    font: 13px/1.2 "Droid Sans Mono",Menlo,Monaco,"Fira Code",Consolas,monospace;
```

Comment:
There are ligature concerns with "Fira Code" in the bug; can we use a simple monospace variant of Fira instead?